imap_processing/lo/l1c/lo_l1c.py

```diff
@@ -56,6 +58,24 @@ def lo_l1c(sci_dependencies: dict, anc_dependencies: list) -> list[xr.Dataset]:
         l1b_goodtimes_only = filter_goodtimes(l1b_de, anc_dependencies)
         pset = initialize_pset(l1b_goodtimes_only, attr_mgr, logical_source)
         full_counts = create_pset_counts(l1b_goodtimes_only)
+
+        # Set the pointing start and end times based on the first epoch
```

Comment:
Can we use the pointing number instead? These products are all going to be built from products with a repointing number flag in it. So would it make sense to flow that through and do the lookup based on pointing number?

I think this is related to: https://github.com/IMAP-Science-Operations-Center/imap_processing/issues/1733
Is there a reason why we aren't using the interpolate_spin_data function like mentioned in that comment? I think your function below is somewhat duplicating that effort.

Reply:
I forgot about the pointing number. I don't have that in my file names yet, but that would be better to use. 

I also forgot about that ticket. However, the interpolate_repoint_data uses MET times and I only have TTJ2000 at this point. I don't think we have function to convert from TTJ2000 to MET.